src/nvim/eval/typval.c

```diff
@@ -2042,11 +2042,14 @@ bool tv_islocked(const typval_T *const tv)
 ///
 /// @param[in]  lock  Lock status.
 /// @param[in]  name  Variable name, used in the error message.
-/// @param[in]  name_len  Variable name length.
+/// @param[in]  name_len  Variable name length. Use #TV_TRANSLATE to translate
+///                       variable name and compute the length. Use #TV_CSTRING
+///                       to compute the length with strlen() without
+///                       translating.
```

Comment:
Maybe clarify more by saying that constants are used to defer gettext translation and length calculation for performance since the name is only used for error messages.